BuildNotifications/Resources/Global/SearchTextBoxStyles.xaml

```diff
@@ -0,0 +1,226 @@
+                                        <Border BorderBrush="{DynamicResource Background3}">
+                                            <ContentPresenter
+                                                Content="{Binding SearchCriteriaViewModel, RelativeSource={RelativeSource AncestorType={x:Type search:SearchTextBox}, Mode=FindAncestor}}" />
+                                        </Border>
+                                    </StackPanel>
+                                </Border>
+                            </Popup>
+                        </Grid>
+
+                        <i:Interaction.Triggers>
+                            <i:EventTrigger EventName="GotKeyboardFocus">
+                                <text:FontSizeChange TargetSize="{StaticResource Block}"
+                                                     Target="{Binding ElementName=Label}" Duration="0.1" />
+                            </i:EventTrigger>
+                            <i:EventTrigger EventName="LostKeyboardFocus">
+                                <text:FontSizeChange TargetSize="{StaticResource BlockFiveSixths}"
+                                                     Target="{Binding ElementName=Label}" Duration="0.35" />
+                            </i:EventTrigger>
+                        </i:Interaction.Triggers>
+                    </Border>
+                    <ControlTemplate.Triggers>
+                        <DataTrigger Binding="{Binding Type, ElementName=Icon}" Value="None">
+
+                            <Setter Property="Visibility" TargetName="Icon" Value="Collapsed" />
+                        </DataTrigger>
+                        <Trigger Property="IsEnabled" Value="false">
+                            <Setter Property="Opacity" TargetName="border" Value="0.56" />
+                        </Trigger>
+                        <Trigger Property="IsMouseOver" Value="true">
+                            <Setter Property="BorderBrush" TargetName="border"
+                                    Value="{DynamicResource Foreground1HalfTransparency}" />
+                            <Setter Property="Foreground" TargetName="Icon"
+                                    Value="{DynamicResource Foreground1HalfTransparency}" />
+                            <Setter Property="Foreground" TargetName="Label"
+                                    Value="{DynamicResource Foreground1HalfTransparency}" />
+                        </Trigger>
+                        <Trigger Property="IsFocused" Value="true">
+                            <Setter Property="BorderBrush" TargetName="border" Value="{DynamicResource Foreground1}" />
+                            <Setter Property="Foreground" TargetName="Icon" Value="{DynamicResource Foreground1}" />
+                            <Setter Property="Foreground" TargetName="Label" Value="{DynamicResource Foreground1}" />
+                            <Setter Property="MaxHeight" Value="9999" />
```

Comment:
Is this just a "big enough number that should suffice" or has it some other meaning?

Reply:
This is a "I tested something and forgot to remove it"